nautobot/core/views/generic.py

```diff
@@ -294,7 +301,7 @@ def get(self, request):
             # Construct the objects table
             # Order By is needed in the table `__init__` method
             order_by = self.request.GET.getlist("sort")
```

Comment:
Can we do the `order_by` check here too instead of having the `hide_hierarchy` related checks in 2 different places?

```py
if order_by:
    hide_hierarchy_ui=True
```